Utilities/FrameworkResources/exports.exp

```diff
@@ -41,6 +41,7 @@ _HCHttpCallRequestGetRequestBodyReadFunction
 _HCHttpCallResponseSetResponseBodyWriteFunction
 _HCHttpCallResponseGetResponseBodyWriteFunction
 _HCHttpCallResponseSetGzipCompressed
+_HCHttpCallRequestSetProgressReportFunction
```

Comment:
the two .exp files should have the same edits as the as the .def files in the \build folder

Reply:
that's right, otherwise we may see issues in iOS

Reply:
Alright, I just added the missing calls